packages/test-utils/src/wrapper.js

```diff
@@ -531,6 +531,81 @@ export default class Wrapper implements BaseWrapper {
     this.vnode = this.vm._vnode
   }
 
+  /**
+   * Sets element value and triggers input event
+   */
+  setValue (value: any) {
+    const el = this.element
+
+    if (!el) {
+      throwError('cannot call wrapper.setValue() on a wrapper without an element')
+    }
+
+    const tag = el.tagName
+    const type = this.attributes().type
+    const event = 'input'
+
+    if (this.isVueComponent) {
+    } else if (tag === 'SELECT') {
+      // event = 'change'
+      // el.value = value
+      throwError('wrapper.setValue() cannot be called on select')
+    } else if (tag === 'INPUT' && type === 'checkbox') {
+      throwError('wrapper.setValue() cannot be called on checkbox. Use wrapper.setChecked() instead')
+    } else if (tag === 'INPUT' && type === 'radio') {
+      throwError('wrapper.setValue() cannot be called on radio. Use wrapper.setChecked() instead')
+    } else if (tag === 'INPUT' || tag === 'textarea') {
+    } else {
+      throwError('wrapper.setValue() cannot be called on this element')
+    }
+
+    el.value = value
+    this.trigger(event)
+  }
+
+  /**
+   * Checks radio button or checkbox element
+   */
+  setChecked (checked: boolean) {
+    console.log(typeof checked)
+    if (typeof checked !== 'undefined') {
+      if (typeof checked !== 'boolean') {
+        throwError('wrapper.setChecked() must be passed a boolean')
+      }
+    } else {
+      checked = true
+    }
+
+    const el = this.element
+
+    if (!el) {
+      throwError('cannot call wrapper.setChecked() on a wrapper without an element')
+    }
+
+    const tag = el.tagName
+    const type = this.attributes().type
+    const event = 'change'
+
+    if (this.isVueComponent) {
+      throwError('wrapper.setChecked() cannot be called on component. Use wrapper.setValue() instead')
```

Comment:
A component instance still has a root element, which can have its value set, so I think we should let setChecked work for vue instance wrappers

Reply:
My concern is related to how v-model behaves on components, but Im not sure. Let me know your final decision

Reply:
Can you remove this check for isVueComponent